### PR TITLE
Updated alm-exaple in CSV

### PR DESCRIPTION
**Pull request by slintes**

### CODE REVIEW

File: deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,24 +12,24 @@ metadata:
           },
           "spec": {
             "cpu": {
-              "isolated": "2-3",
+              "isolated": "1-3",
```
Comment (MarSik) [inline deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: Isolated and reserved must not overlap. Also for now, making nonIsolated the same as reserved is probably the best approach. In the future nonIsolated will be the same as all cpus - isolated and reserved will be subset of nonIsolated.

Comment (slintes) [reply deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: ok, thx, fixed

**Comment by fromanirh** [comment]
Looks like issue is in CI:
```could not wait for build: the build root failed after 55s with reason DockerBuildFailed: Docker build strategy has failed.

Fedora 31 - x86_64                               11 MB/s |  71 MB     00:06    
No match for argument: golang-1.13.5-1.fc31
Package tar-2:1.32-2.fc31.x86_64 is already installed.
Error: Unable to find a match: golang-1.13.5-1.fc31
error: build error: running 'dnf -y install     $GOVERSION...ext     which     && dnf clean all' failed with exit code 1```

**Comment by slintes** [comment]
> Looks like issue is in CI:
> 
> ```
> 
> Fedora 31 - x86_64                               11 MB/s |  71 MB     00:06    
> No match for argument: golang-1.13.5-1.fc31
> Package tar-2:1.32-2.fc31.x86_64 is already installed.
> Error: Unable to find a match: golang-1.13.5-1.fc31
> error: build error: running 'dnf -y install     $GOVERSION...ext     which     && dnf clean all' failed with exit code 1```
> ```

yes, fix is on its way already in #51

**Comment by davidvossel** [comment]
/hold

this logic needs to be codified and go into #48 once it is merged. 

We can inject whatever alm examples we want inline within that generation logic.

**Comment by davidvossel** [comment]
also, this won't pass CI. There's logic in the `make ci-job` make target to ensure we never make manual changes to a CSV file. It will be detected that these changes aren't auto generated and fail.

**Comment by fromanirh** [comment]
> /hold
> 
> this logic needs to be codified and go into #48 once it is merged.
> 
> We can inject whatever alm examples we want inline within that generation logic.

Ack, I didn't realize this.

**Comment by slintes** [comment]
Ouch, yes, I missed that the example is coming from `deploy/crds/performance.openshift.io_v1alpha1_performanceprofile_cr.yaml`

Closing for now, will create new PR if needed after #48 is merged

/close

**Pull request closed without merging**